app/repositories/story_repository.rb

```diff
@@ -1,7 +1,10 @@
+require_relative "../helpers/url_helpers"
 require_relative "../models/story"
 require_relative "../utils/sample_story"
 
 class StoryRepository
+  extend UrlHelpers
```

Comment:
Maybe failing test could be due to `extend` vs `include`? Just a stab in the dark :)